draft-js-mention-plugin/src/MentionSuggestions/index.js

```diff
@@ -261,6 +262,19 @@ export default class MentionSuggestions extends Component {
     }
 
     const { theme = {} } = this.props;
+    const groups = this.props.suggestions.reduce((data, value) => {
+      const type = value.get('type') || 'other';
+      let items = data.get(type);
+      if (!items) {
+        items = Immutable.OrderedSet();
+      }
+      items = items.add(value);
+      return data.set(type, items);
+    }, Immutable.OrderedMap());
+
+    const groupTitles = Object.assign({ other: 'Other' }, this.props.groupTitles);
```

Comment:
can you use the { ...this.props.groupTitles, other: 'Other' } instead of Object.assign?